sdk/playwrighttesting/microsoft-playwright-testing/review/microsoft-playwright-testing-reporter.api.md

```diff
@@ -0,0 +1,24 @@
+## API Report File for "@azure/microsoft-playwright-testing"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import type { FullConfig } from '@playwright/test/reporter';
+import type { FullResult } from '@playwright/test/reporter';
+import type { Reporter } from '@playwright/test/reporter';
+import type { Suite } from '@playwright/test/reporter';
+import type { TestCase } from '@playwright/test/reporter';
+import type { TestResult } from '@playwright/test/reporter';
+
+// @public
+class MPTReporter implements Reporter {
+    constructor(obj: Partial<MPTReporterConfig>);
```

Comment:
instead of `obj` can we call the param `config` or `configuration`?
```suggestion
    constructor(config: Partial<MPTReporterConfig>);
```

In this case `config` might be the better term as it seems consistent with the playwright naming convention